src/test/java/com/elasticbox/jenkins/tests/PullRequestLifecycleManagementTest.java

```diff
@@ -148,7 +149,7 @@ private void testSyncPayload(String sha) throws IOException {
     }
 
     private void ensureBuildTriggered(String messageFormat, Object parameter) {
-        final AbstractBuild build = waitForNextBuild(60);
+        final AbstractBuild build = waitForNextBuild(60, "ensureBuildTriggered");
```

Comment:
Maybe this "60" could be a more reasonable timeout, and could be unified with the constant already defined